### ISSUE TITLE
Is there already gpu support? 

### ISSUE THREAD

**Issue by loeffel-io** [giongto35/cloud-morph#61]

**Comment by giongto35** [comment]
: D, yes it's not ready. I'm not able to research it too because of the high cost of Cloud GPU machine :(

**Comment by toby63** [comment]
I guess that Xvfb does not support GPU acceleration.

An alternative, which is used by x11docker (https://github.com/mviereck/x11docker/wiki/X-server-and-Wayland-Options#attributes-of-x-server-and-wayland-options), would be:
weston + xwayland + xdotool

Quote from link above:

> With option --gpu a setup with Weston, Xwayland and xdotool is used (instead of Xdummy or Xvfb) and runs on X only.

**Comment by loeffel-io** [comment]
Any updates on this? ✌️

**Comment by sergystepanov** [comment]
Nope. But there are [some](https://telegra.ph/NVIDIA-GPU-cloud-morph-12-09) attempts.
